internal/cmd/mongodbflex/backup/schedule/schedule.go

```diff
@@ -0,0 +1,151 @@
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read MongoDB Flex instance: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, resp.Item)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "Instance ID")
+
+	err := flags.MarkFlagsRequired(cmd, instanceIdFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		InstanceId:      *flags.FlagToStringPointer(p, cmd, instanceIdFlag),
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *mongodbflex.APIClient) mongodbflex.ApiGetInstanceRequest {
+	req := apiClient.GetInstance(ctx, model.ProjectId, model.InstanceId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, instance *mongodbflex.Instance) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		output := struct {
+			BackupSchedule                 string `json:"backup_schedule"`
+			DailySnaphotRetentionDays      string `json:"daily_snapshot_retention_days"`
+			MonthlySnapshotRetentionMonths string `json:"monthly_snapshot_retention_months"`
+			PointInTimeWindowHours         string `json:"point_in_time_window_hours"`
+			SnapshotRetentionDays          string `json:"snapshot_retention_days"`
+			WeeklySnapshotRetentionWeeks   string `json:"weekly_snapshot_retention_weeks"`
+		}{
+			BackupSchedule:                 *instance.BackupSchedule,
+			DailySnaphotRetentionDays:      (*instance.Options)["dailySnapshotRetentionDays"],
+			MonthlySnapshotRetentionMonths: (*instance.Options)["monthlySnapshotRetentionDays"],
+			PointInTimeWindowHours:         (*instance.Options)["pointInTimeWindowHours"],
+			SnapshotRetentionDays:          (*instance.Options)["snapshotRetentionDays"],
+			WeeklySnapshotRetentionWeeks:   (*instance.Options)["weeklySnapshotRetentionWeeks"],
+		}
+		details, err := json.MarshalIndent(output, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal MongoDB Flex backup schedule: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.AddRow("BACKUP SCHEDULE", *instance.BackupSchedule)
+		table.AddSeparator()
+		table.AddRow("DAILY SNAPSHOT RETENTION DAYS", (*instance.Options)["dailySnapshotRetentionDays"])
```

Comment:
Maybe in the title we can write as "...snapshot retention period (DAYS)", same for the other columns.

Reply:
Yes, this was the intention but I somehow forgot about it. Thanks 😁